.github/workflows/self-scheduled.yml

```diff
@@ -252,7 +252,7 @@ jobs:
     runs-on: ${{ matrix.machines }}
     container:
       image: huggingface/transformers-tensorflow-gpu
-      options: --gpus 0 --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
+      options: --gpus all --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
```

Comment:
set to `all` here, because  we have `machines: [multi-gpu-docker, single-gpu-docker]`